sensor_msgs/Image rgb
--- 
object_keypoints_ros/Keypoints detection
